### ISSUE TITLE
issue with jupyter notebooks

### ISSUE THREAD

**Issue by ltalirz** [DropD/reentry#27]
@yakutovicha ran into an issue on jupyter.materialscloud.org, where people can either go to the command line and issue python/verdi commands or they can connect to a jupyter notebook server.

When checking on the command line via ipython, `reentry` worked fine:
```
ipython

In [1]: import reentry

In [2]: from reentry import manager as epm

In [3]: [ep for ep in epm.iter_entry_points("aiida.calculations")]
Out[3]:
[reentry.EntryPoint.parse(codtools.ciffilter = aiida_codtools.calculations.ciffilter:CiffilterCalculation),
 reentry.EntryPoint.parse(codtools.cifcellcontents = aiida_codtools.calculations.cifcellcontents:CifcellcontentsCalculation),
 reentry.EntryPoint.parse(codtools.cifcoddeposit = aiida_codtools.calculations.cifcoddeposit:CifcoddepositCalculation),
...]
```

However, when executing the same commands in the jupyter notebook, the list `[ep for ep in epm.iter_entry_points("aiida.calculations")]` is empty.

If one then executes `! reentry scan -r aiida` *inside* the jupyter notebook, the reentry points are populated again (note: executing this on the regular command line has no effect).

We checked that we are using the same reentry module (same physical location on disk) inside the ipython shell and inside the jupyter notebook.
@DropD Do you have an idea why reentry might be using a different cache, when inside the jupyter notebook?

Edit: This is using reentry 1.2.0

**Comment by DropD** [comment]
This might be related to closed issue #25. Could you please test with reentry v1.2.1a2 to find out? If that's the problem, and v1.2.1a2 solves it, I will release it as v1.2.1.

**Comment by yakutovicha** [comment]
Just tested it with the reentry v1.2.1a2. 

Everything seems to work now.

I guess the issue may be closed.
